index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>GifTastic</title>
+
+    <!-- bootstrap if i wanted to use it -->
+    <!-- css style sheet link -->
+    
```

Comment:
A div here to have buttons and maybe a div to display the gifs would be very useful